test/unquote-properties-test.js

```diff
@@ -0,0 +1,13 @@
+var x = {
+  'quotedProp': 1,
+  unquotedProp: 2,
+  'quoted-prop': 3,
+  method() { return 4; },
```

Comment:
Did not even know you could do that! I'll add the test, but I don't think I'll handle the case.